src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -325,23 +325,23 @@ FOR_ALL_NUMA_FUNCTIONS
         }
         else
         {
-            g_numaAvailable = true;
-            g_highestNumaNode = numa_max_node();
+            g_numaAvailable_gc = true;
+            g_highestNumaNode_gc = numa_max_node();
         }
     }
 #endif // HAVE_NUMA_H
-    if (!g_numaAvailable)
+    if (!g_numaAvailable_gc)
     {
         // No NUMA
-        g_highestNumaNode = 0;
+        g_highestNumaNode_gc = 0;
     }
 }
 
 // Cleanup of the NUMA support data structures
 void NUMASupportCleanup()
 {
 #if HAVE_NUMA_H
-    if (g_numaAvailable)
+    if (g_numaAvailable_gc)
```

Comment:
Better way to fix collisions like this one is to make the variable `static`. It will make it invisible from the current compilation unit.

Reply:
thanks, updated. Also using anonymous namespace now to reduce renaming.